package_test.go

```diff
@@ -235,9 +234,44 @@ func TestDocumentsClientCreate(t *testing.T) {
 	}{}
 	require.NoError(t, json.Unmarshal([]byte(testResponse), expected))
 	assert.Equal(t, &expected.Data, got)
-}  
-  
-  
+}
+
+func TestDocumentsClientUpdate(t *testing.T) {
+	testResponse := exampleDocumentsResponse_1documents
+
+	// Prepare HTTP client with mocked transport.
+	hc := &http.Client{}
+	hc.Transport = &testutils.MockRoundTripper{RoundTripFn: func(r *http.Request) (*http.Response, error) {
+		// Assert request method and URL.
+		assert.Equal(t, http.MethodPost, r.Method)
+		u, err := url.JoinPath(testBaseURL, common.DocumentsUpdateEndpoint())
+		require.NoError(t, err)
+		assert.Equal(t, u, r.URL.String())
+
+		testAssertHeaders(t, r.Header)
+		testAssertBody(t, r, fmt.Sprintf(`{"id":"%s", "title":"%s", "text":"%s", "publish":%t}`, "497f6eca-6276-4993-bfeb-53cbbbba6f08", "🎉 Welcome to Acme Inc", "Updated text!", true))
+
+		return &http.Response{
+			Request:       r,
+			ContentLength: -1,
+			StatusCode:    http.StatusOK,
+			Body:          io.NopCloser(strings.NewReader(testResponse)),
+		}, nil
+	}}
+
+	cl := outline.New(testBaseURL, hc, testApiKey)
+	var id outline.DocumentID = "497f6eca-6276-4993-bfeb-53cbbbba6f08"
+	got, err := cl.Documents().Update(id).Title("🎉 Welcome to Acme Inc").Text("Updated text!").Publish(true).Do(context.Background())
```

Comment:
Why isn't this inlined? 🤔 
```suggestion
	got, err := cl.Documents().Update(outline.DocumentID("497f6eca-6276-4993-bfeb-53cbbbba6f08")).Title("🎉 Welcome to Acme Inc").Text("Updated text!").Publish(true).Do(context.Background())
```